Adding Jim's profile (#519)
SSWConsulting/SSW.People.Profiles: Jim-Zheng/Jim-Zheng.md

---
id: "187c0052-f4bc-eb11-bacc-00224814a68b"

qualifications: ""
category: "Developers"

> role: "Sample Profile"

Reviewer: delete category and role

quote: "Think like a man of action, act like a man of thought."
quoteAuthor: "Henri Bergson"
---

> [Editing your profile]: https://github.com/SSWConsulting/People/wiki/3.-Editing-your-profile

Reviewer: Delete edit profile link

> Jim Zheng is a software developer based in SSW Hangzhou office. With more than 3 years of experience in web application development, he has worked as a technical lead and involved in multiple projects with complex requirements and various structures, which including course management system, geotechnical experiment system and electrical monitoring system, gaining extensive experience on .NET based technologies and in-depth understanding of project design.

Reviewer: I recommend this wording:

```suggestion
Jim Zheng is a software developer based in the SSW Hangzhou office. With more than 3 years of experience in web application development, he has worked as a technical lead and been involved in multiple projects with complex requirements and varying structures. He has worked on various products, including a course management system, a geotechnical experiment system and an electrical monitoring system. Through these projects, he has gained extensive experience with .NET based technologies and an in-depth understanding of project design.
```

> He has been continuously chasing the best practice for designing robust, applicable and maintainable projects by leveraging outstanding methodologies such as domain driven design, with the ultimate aim for minimum cost and maximum product lifetime. He is also passionate at digging into systems to understanding how things work.

Reviewer: I recommend this wording:

```suggestion
He has been continuously striving to understand the best practices for designing robust, applicable and maintainable projects by leveraging outstanding methodologies such as domain-driven design, with the ultimate aim of minimum cost and maximum product lifetime. He is also passionate about digging into systems to understanding how things work.

```
